test/spatial_reaction_systems/lattice_reaction_systems_ODEs.jl

```diff
@@ -12,6 +11,9 @@ include("../spatial_test_networks.jl")
 using StableRNGs
 rng = StableRNG(12345)
 
+# Sets defaults
```

Comment:
The spatial tests are already good. Further changes here will wait until that  part is done (the internal branches I have already do some modifications)